src/cryptoadvance/specter/server_endpoints/devices.py

```diff
@@ -40,6 +42,8 @@ def new_device_type():
 @devices_endpoint.route("/new_device_keys/<device_type>/", methods=["GET", "POST"])
 @login_required
 def new_device_keys(device_type):
+    device_class = get_device_class(device_type)
+    template = device_class.template
```

Comment:
```suggestion
    template = get_device_class(device_type).template
```

Reply:
But, then we'd have no `device_class` variable to pass in `render_template()`, no?

Reply:
Sorry, was too quick with that.